tools/cpp/BUILD.windows.tpl

```diff
@@ -215,6 +222,205 @@ toolchain(
     toolchain_type = "@bazel_tools//tools/cpp:toolchain_type",
 )
 
+cc_toolchain(
+    name = "cc-compiler-x64_x86_windows",
+    toolchain_identifier = "msvc_x64_x86",
+    toolchain_config = ":msvc_x64_x86",
+    all_files = ":empty",
+    ar_files = ":empty",
+    as_files = ":msvc_compiler_files",
+    compiler_files = ":msvc_compiler_files",
+    dwp_files = ":empty",
+    linker_files = ":empty",
+    objcopy_files = ":empty",
+    strip_files = ":empty",
+    supports_param_files = 1,
+)
+
+cc_toolchain_config(
+    name = "msvc_x64_x86",
+    cpu = "x64_windows",
```

Comment:
I think this should be "x64_x86_windows" ?

Reply:
I will need to change this method `_use_msvc_toolchain` in `windows_cc_toolchain_config.bzl`, to use MSVC toolchains for the new cpu values right?

Reply:
Oh, I see. This is actually not affecting the toolchains selection, but only used in `windows_cc_toolchain_config.bzl`. Since other config like `msys_x64` also uses `x64_windows`, let's just keep it the same here.